### ISSUE TITLE
[Accessibility] wixtoolset does not offer alternative text to WixUI_Bmp_Dialog and WixUI_Bmp_Banner image

### ISSUE THREAD

**Issue by pengzhencao** [wixtoolset/issues#7398]
* Which version of WiX are you building with?
wix-3.11.1

* Which version of Visual Studio are you building with (if any)?
vs2013 u3

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
WixUIExtension

* Which version of .NET are you building with?
N/A

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?
win10 enterprise

* Describe the problem and the steps to reproduce it.
1. Prepare the NVDA screen reader ready
2. Run the GUI installer built with Wixtoolset(here it is VMware Tools installer for windows)
3. Following the installation wizard, select Complete install type, click next and wait for the pop up window. (Or in typical install)

* Describe the behavior you expected and how it differed from the actual behavior.
Screen reader does not read the Banner image, which may cause difficulty to users rely screen reader.
<img width="445" alt="VPAT17966issuescreenshotdda8It" src="https://user-images.githubusercontent.com/17611271/231695945-4ae7ab24-361c-4bf3-ac0c-c149e12e35aa.png">

**Comment by AlasdairKing** [comment]
**Users affected**
This potentially impacts people with sight impairments: dead and Deaf people, people with cognitive impairments, people with physical impairments are all unaffected. 

People with limited sight impairments (e.g. using a screen magnifier) are also unaffected: this potentially impacts only people with sight impairments using screenreaders (e.g. JAWS, Narrator) as their main AT. 

**Analysis based on users**
The logo is **pure decoration**: the logo content ("VMWare") is communicated in the screen title ("Ready to install VMWare Tools"). It should not be communicated to screenreader users since this simply slows down their operation of the dialog, making it less accessible (because it takes them longer to operate it: it's a superfluous and redundant piece of text scroll through to get to the button. 

This isn't a web app, but the WCAG guidelines is helpful for reference: 
https://www.w3.org/TR/WCAG21/#non-text-content

@pengzhencao is to be praised for their thoughtfulness, but adding "alt text" for this image would make the dialog less usable, not more. 🙂

**Comment by pengzhencao** [comment]
Ok. Thanks for the explanation and I think you could close this issue

**Comment by jmjaffe37** [comment]
@robmen @barnson, would it be possible to reopen this issue? There is a different use-case for setting the names of banner images:

In the browseDlg (gotton to by clicking the `browse` button in the CustomizeSetup Dialog), the two icons for going `up one level` or `create a new folder` have the name properties `WixUI_Bmp_Dialog` and `WixUI_Bmp_Banner` by default. This is a problem for accessibility since the names are non-descriptive. When changing the names (via a loc string setting), this causes the icons to be replaced by text

![image](https://github.com/user-attachments/assets/9b7be22b-532b-4776-a83a-26f8040785d6)

**Comment by AlasdairKing** [comment]
@jmjaffe37 identifies a definite problem, but it isn't the banner image(s). The controls in this dialog are not correctly labelled. They are labelled with confusing names referencing the bitmaps, but it's not the bitmap that's at fault.

- The PushButton control "WixUI_Bmp_Up" has a Text property of "WixUI_Bmp_Up" but it should have a Text property of "Up a level" 
- The PushButton control "NewFolder" has a Text property of "WixUI_Bmp_New" but it should have a Text property of "New folder"

There are some other problems, but the above will fix this immediate problem. 
- The Text control "PathLabel" has a Control_Next property of "BannerBitmap" but it should have a Control_Next property of "PathEdit"... oh, but then there is a cycle.

**Comment by AlasdairKing** [comment]
Oh, I see: in the image below the icons on the buttons have disappeared, replaced by the text I suggested. This is is not usable for a sighted user since the text does not fit on the buttons. We would have to resize the buttons and no longer have images on them:

![image](https://github.com/user-attachments/assets/d3ed4669-ac35-40d3-8eea-dc712fcd959f)
